dipy/reconst/dki.py

```diff
@@ -966,7 +1236,7 @@ def __init__(self, gtab, fit_method="OLS", *args, **kwargs):
 
         References
         ----------
```

Comment:
Watch the typo in line 1200:
# lopping for all voxels
# looping for all voxels
or as stated in other parts of the code
# loop over all voxels